### PR TITLE
feat[sidebar] create sidebar and change icon, title page

### PR DESCRIPTION
**Pull request by javonunezz**
### Context
In order to allow the user to navigate through the different pages, a sidebar is created with its respective buttons that allow the user to navigate.

### What did I do?
I have created a sidebar with several buttons and changed the title of the page and the icon.
The Sidebar have a design sm and md.

#### design md
image
#### Title and icon changed
image
#### design sm
image

### CODE REVIEW

File: src/components/Organisms/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import { IoIosSchool } from "react-icons/io";
+import { IoMdSettings } from "react-icons/io";
+import { CiLogout } from "react-icons/ci";
+import { AiFillHome } from "react-icons/ai";
+
+export const Sidebar = ({ log }) => {
+  return (
+    <div className="min-h-16 flex flex-row items-center justify-center gap-x-7 bg-primary-500 py-3 md:h-screen md:w-64 md:flex-col md:items-center md:justify-center md:gap-x-0 md:space-y-5 md:px-8">
+      <Button
+        icon={<AiFillHome />}
```
Comment (yurehn) [inline src/components/Organisms/Sidebar.jsx]: These lines are repeated:
color="bg-primary-500"
position={"justify-around"}
iconColor={"white"}
iconSize={"1.25em"}
sizeSM_w={"w-10"}
sizeSM_h={"h-10"}
sizeMD_w={"md:w-36"}
sizeMD_h={"md:h-14"}

ver posibilidad de refactorizar esta parte.

Comment (javonunezz) [reply src/components/Organisms/Sidebar.jsx]: 👎

Comment (alegrimminck) [reply src/components/Organisms/Sidebar.jsx:12]: > 

Sorry cant understando your palabrations

**Review by yurehn — CHANGES_REQUESTED** [review]
review/correct comments added 👍

File: src/components/Organisms/Sidebar.jsx (hunk 1 above)
Comment (yurehn) [reply src/components/Organisms/Sidebar.jsx]: Oh, sorry for that my friend

**Comment by alegrimminck** [comment]
Se ve bonito, buen arreglo :+1:  aprovado

**Review by alegrimminck — APPROVED** [review]
Bueno bueno, siga así :+1: aprovado

**Review by yurehn — APPROVED** [review]
Excellent 😀👍

**Pull request merged**